python/aida_interchange/ldc_time_component.py

```diff
@@ -1,7 +1,7 @@
 from enum import Enum
 from rdflib import Graph, Literal, XSD, URIRef
-from aida_interchange.aida_rdf_ontologies import AIDA_ANNOTATION
-from aida_interchange import aifutils
+from aida_rdf_ontologies import AIDA_ANNOTATION
+import aifutils
```

Comment:
I apologize I didn't comment this on the original PR, but from what I have noticed in python code it looks like it is best practice to place all `imports` at the top and `from` statements after the imports.